### DISCUSSION TITLE
请问如何选定优化器和学习率还有其他参数

### DISCUSSION THREAD

**Discussion by primising** [RUCAIBox/RecBole#1840] [Q&A]
我尝试在官网的示例中的config直接添加config = Config(model='LightGCN', dataset='ml-100k', learner='adam')但是会报错TypeError: __init__() got an unexpected keyword argument 'learner'，还有显示的超参是训练300轮，第75轮就停下了是因为有设置loss不在降低就停止吗。我想将训练中的数据绘制成图该如何做。

**Comment by VladHardAndSoftWare** [comment]
You can use the built-in hyperparameter selection to find the most suitable parameters for your model. And also display the matrices in the form of a graph for various model parameters. https://recbole.io/docs/user_guide/usage/parameter_tuning.html

**Reply by primising** [reply]
Now I can use config to set parameters. In your link I can't to use parameter tuning, you have the complete code for use parameter tuning?
Thanks.
